app/helpers/turbo/streams/action_helper.rb

```diff
@@ -9,13 +11,13 @@ module Turbo::Streams::ActionHelper
   #
   #   turbo_stream_action_tag "replace", targets: "message_1", template: %(<div id="message_1">Hello!</div>)
   #   # => <turbo-stream action="replace" targets="message_1"><template><div id="message_1">Hello!</div></template></turbo-stream>
-  def turbo_stream_action_tag(action, target: nil, targets: nil, template: nil)
-    template = action.to_sym == :remove ? "" : "<template>#{template}</template>"
+  def turbo_stream_action_tag(action, target: nil, targets: nil, template: nil, **attributes)
+    template = action.to_sym == :remove ? "" : tag.template(template.to_s.html_safe)
```

Comment:
Could `tag.template(template.to_s.html_safe)` be replaced by `tag.template(template)`? 

I'm able to render this without issue:

```erb
<%= tag.template(nil) %>
```

Reply:
That would be finr for the nil/empty case, but if there is a template it's going to be escaped.

Reply:
Ah, I understand now. Thank you for clarifying.

Since we're expanding the interface of this helper, how do you feel about also accepting the `template` value as a block, like most other Action View tag helpers?

```ruby
template = block.present? ? capture(&block) : template.to_s.html_safe
# ...
```

Reply:
I was also thinking about that. I first looked at how to extend the [`TagBuilder#action` method](https://github.com/hotwired/turbo-rails/blob/main/app/models/turbo/streams/tag_builder.rb#L214-L218) which would already support that, but since it's already accepting the `rendering` arguments for rendering the template I figured it would be simpler to extend the `turbo_stream_action_tag` with just the additional attributes.

Or do you see a non-hacky way to adjust the `action` method to take additional attributes?

I don't think this is a good idea:
```diff
  def action(name, target, content = nil, allow_inferred_rendering: true, **rendering, &block)
    template = render_template(target, content, allow_inferred_rendering: allow_inferred_rendering, **rendering, &block)
   
-    turbo_stream_action_tag name, target: target, template: template
+    turbo_stream_action_tag name, target: target, template: template, **rendering
  end
```